index.html

```diff
@@ -96,9 +114,10 @@ <h3 class="card-main-title">Profesional Art Printing Data</h3>
                         </section> 
                         <button type="button" class="card-big-btn" >See project</button>    
                     </section>
-                    <section class="post">
+                    <section class="post post3">
                         <section class="card-contents">
                             <h3 class="card-main-title">Profesional Art Printing Data</h3>
+                            <h3 class="card-main-title1">Website Portfolio </h3>
                             <p class="card-description">A daily selection of privately personalized reads;
                                no accounts or sign-ups required. Has been the industry's standard.
                             </p>
```

Comment:
Great job implementing a hover interaction on work sections card 👍 🎖️ 

- [ ] Please make sure that when a card is hovered the project title, description, and list of technology image disipear and the project image appeard on front. 
**Your project hover interaction**
![image](https://user-images.githubusercontent.com/74411135/229297268-7b371e0f-f22b-457c-9bb8-a32229b4d4b6.png)

**Figma design hover interaction**
![image](https://user-images.githubusercontent.com/74411135/229297291-b786bc42-7faf-4924-9945-eff7a613abf5.png)

This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪